conf/modules.config

```diff
@@ -268,7 +268,7 @@ process {
         // one run with multiple samples, so fix ID to just db name to ensure clean log name
         ext.prefix = { "${meta.db_name}" }
         publishDir = [
-            path: { "${params.outdir}/malt/${meta.db_name}" },
+            path: { "${params.outdir}/malt/${meta.db_name}/" },
```

Comment:
Are all of these changes, appending `/` to the path, really required or is it just a style thing?